src/fe/fe_l2_lagrange_shape_3D.C

```diff
@@ -1248,9 +1248,10 @@ Real FE<3,L2_LAGRANGE>::shape_second_deriv(const ElemType type,
                              << std::endl;
               warning_given_HEX20 = true;
             }
+            /* FALLTHROUGH */
 
-            // triquadraic hexahedral shape funcions
           case HEX27:
+            // triquadraic hexahedral shape funcions
```

Comment:
Hmm... GitHub thinks you changed this line (whitespace?) but I don't see it. Might as well fix the 50% of the words that are misspelled too.

Reply:
Will do; thanks.

I moved that line to after the case statement, in the process of figuring out how loose I could be with those comments and still have the gcc7 warning engine recognize them.  But "fixing multiple typos" is a much better excuse for the edit than "probing the gcc7 comment parser via trial and error".